display_list/geometry/dl_region_unittests.cc

```diff
@@ -309,8 +310,7 @@ TEST(DisplayListRegion, UnionEmpty) {
     DlRegion u = DlRegion::MakeUnion(region1, region2);
     EXPECT_EQ(u.bounds(), SkIRect::MakeEmpty());
     auto rects = u.getRects();
-    std::vector<SkIRect> expected{};
-    EXPECT_EQ(rects, expected);
+    EXPECT_TRUE(rects.empty());
```

Comment:
Also `u.isEmpty()`?

Reply:
(Suggesting to test the new isEmpty method on DlRegion that I didn't realize was private before.)